contracts/protocol/bases/SellerBase.sol

```diff
@@ -142,36 +142,42 @@ contract SellerBase is ProtocolBase, IBosonAccountEvents {
 
         // Map the seller's other addresses to the seller id. It's not necessary to map the treasury address, as it only receives funds
         _lookups.sellerIdByAssistant[_seller.assistant] = _seller.id;
+        _lookups.sellerCreator[_seller.id] = msgSender();
     }
 
     /**
      * @notice Creates a minimal clone of the Boson Voucher Contract.
      *
      * @param _sellerId - id of the seller
      * @param _collectionIndex - index of the collection.
+     * @param _creator - address of the seller creator
      * @param _assistant - address of the assistant
      * @param _voucherInitValues - the fully populated BosonTypes.VoucherInitValues struct
+     * @param _externalId - external collection id ("" for the default collection)
      * @return cloneAddress - the address of newly created clone
      */
     function cloneBosonVoucher(
         uint256 _sellerId,
         uint256 _collectionIndex,
+        address _creator,
         address _assistant,
-        VoucherInitValues calldata _voucherInitValues
+        VoucherInitValues calldata _voucherInitValues,
+        string memory _externalId
```

Comment:
keen to understand why you went for a string here :)

Reply:
For the seller's convenience, I guess. 
I name the first collection "cats", the second "dogs", the third "frogs" etc.

It's not used as a key anywhere, so it's not that problematic.

But yes, we can make it bytes32 and be more efficient.

Reply:
thanks, no need to change, similar to the "getting the name for the ERC20s" i would have just used identifiers and given that responsibility to the clients ... 

In this instance, i would have said that the clients would have had to get the name from the metadata file, and the protocol woudl have only known about IDs .. but yeah, thanks